kai/rpc_server/server.py

```diff
@@ -282,6 +282,15 @@ def initialize(
             ],
         )
 
+        app.log.info("Priming analyzer cache with default run")
```

Comment:
can the messaging be "Running analysis" instead?

Reply:
@pranavgaikwad you mean change `Priming analyzer cache with default run` -> `Running analysis` ?

Reply:
yeah